.github/workflows/native-ubuntu.yml

```diff
@@ -0,0 +1,146 @@
+name: Native Ubuntu
+
+on:
+  push:
+    branches: [ main, ci-sandbox ]
+  pull_request:
+    branches: [ main, ci-sandbox ]
+
+jobs:
+  build-ubuntu:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "full-marian"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            cmake: 
+              CMAKE_BUILD_TYPE: "Release"
+              COMPILE_TESTS: "ON"
+              USE_WASM_COMPATIBLE_SOURCE: "OFF"
+              COMPILE_SERVER: "OFF"
+              COMPILE_EXAMPLES: "OFF"
+              TEST_TAGS: ""
+
+          - name: "full-marian-slim"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            cmake: 
+              CMAKE_BUILD_TYPE: "Slim"
+              COMPILE_TESTS: "ON"
+              USE_WASM_COMPATIBLE_SOURCE: "OFF"
+              COMPILE_SERVER: "OFF"
+              COMPILE_EXAMPLES: "OFF"
+              TEST_TAGS: ""
+
+          - name: "minimal-marian"
+            os: ubuntu-latest
+            gcc: 8
+            cpu: 'ON'
+            gpu: 'OFF'
+            cmake:
+              CMAKE_BUILD_TYPE: "Release"
+              COMPILE_TESTS: "OFF"
```

Comment:
`COMPILE_TESTS` will be `ON` for this. Right?

Additionally, having "minimal-marian" here exactly the same as "minimal-marian" of `native-mac.yml` would be nicer. This one is up to you though 👍

Reply:
Wasn't sure of the implications, `ON` works for me. Thanks for pointing out. Will try to set it accordingly. 

I thought the idea was to just have the bare minimum required - i.e the library for WASM (I'm unsure what works in the minimal marian-path due to the CMake switches).

Reply:
I think if COMPILE_TESTS=ON works for "minimal-marian" then we can keep it ON. One less thing to worry 👍

Reply:
So compile tests seem to be offending sqlite is my conclusion, can I disable them for minimal marian?

Reply:
Go ahead then 👍